### PR TITLE
docs: Added `Project Setup` docs

### PR DESCRIPTION
**Pull request by ajaykumar97**
Added project setup guide to set up the project on a new system.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,35 +1,33 @@
...
 - [Redux Saga](https://github.com/redux-saga/redux-saga) middleware for Redux
-- [Babel](http://babeljs.io/) for ES6+ support
 - [React Navigation](https://reactnavigation.org/) navigation for React Native
 - [React Hooks](https://reactjs.org/docs/hooks-intro.html) for state management in functional components
+- [react-native-config](https://github.com/luggit/react-native-config) for staging and production environments setup
 
 ## Common problems
-  Most likely another process is already using this port
-  Run the following command to find out which process:
+  Most likely another process is already using this port. Run the following command to find out which process:
```
Comment (AbdoHema2016) [inline README.md:15]: Doesn't seem a common problem

File: docs/PROJECT_SETUP.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+2. Switching to the project folder
+  
+      `cd react-native-innovator/Innovator`
+
+3. Installing the `node_modules`
+  
+      `yarn install`
+
+4. Installing the pods (iOS only)
```
Comment (AbdoHema2016) [inline docs/PROJECT_SETUP.md:20]: "cd .. " is not related to installing the pods on the project

File: README.md [hunk 3]
```diff
@@ -1,35 +1,33 @@
 # react-native-innovator
 
 ## Getting Started
-1. Ensure you've followed the [React Native - Get Started Guide](https://reactnative.dev/docs/getting-started.html)
-1. Clone this project `git clone https://github.com/uCreateit/react-native-innovator.git`
-1. Run `npm install` from root directory
-1. Start the app in an emulator
+  Use [this setup guide](docs/PROJECT_SETUP.md) to setup the `Innovator` project on your local machine.
 
 ## Stack
```
Comment (AbdoHema2016) [inline README.md:6]: Appium,Jest & Bitrise should be added to the stack

File: README.md [hunk 4]
```diff
@@ -1,35 +1,33 @@
...
 - [React Native](https://facebook.github.io/react-native/) for building mobile apps using Javascript
 - [Redux](https://redux.js.org/) a predictable state container for Javascript apps
 - [Redux Saga](https://github.com/redux-saga/redux-saga) middleware for Redux
-- [Babel](http://babeljs.io/) for ES6+ support
 - [React Navigation](https://reactnavigation.org/) navigation for React Native
 - [React Hooks](https://reactjs.org/docs/hooks-intro.html) for state management in functional components
+- [react-native-config](https://github.com/luggit/react-native-config) for staging and production environments setup
```
Comment (AbdoHema2016) [inline README.md:12]: U can add the medium article as well to make it easier to understand the setup

File: README.md (hunk 1 above)
Comment (ajaykumar97) [reply README.md:15]: Removing it.

File: docs/PROJECT_SETUP.md (hunk 2 above)
Comment (ajaykumar97) [reply docs/PROJECT_SETUP.md:20]: Yep. It was just to go back to the root of the project. I will remove it.

File: README.md (hunk 3 above)
Comment (ajaykumar97) [reply README.md:6]: Sure. As this commit was only related to the `Project Setup`, so I haven't done much of the stuff in `Readme.md` file In this commit. I will create separate PR to update the Readme.md file

File: README.md (hunk 4 above)
Comment (ajaykumar97) [reply README.md:12]: Sure.

**Comment by ajaykumar97** [comment]
@AbdoHema2016 I have made the requested changes. Please review.

File: docs/PROJECT_SETUP.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+  
+      `cd ios && pod install`
+
+5. Running the builds
+      We can run the android and iOS builds either from the command-line or from the Android Studio and Xcode.
+
+
+    5.1 Running from the command-line
+
+      Use the following scripts(added into the `package.json` file) to run the andorid and iOS apps
```
Comment (Gurpreetgpsingh) [inline docs/PROJECT_SETUP.md:30]: @ajaykumar97 
Please fix typo: from andorid to android

**Pull request merged**